examples/lowMach/lowMach.udf

```diff
@@ -9,13 +9,20 @@
 
 static occa::kernel userQKernel;
 static occa::kernel userVpKernel;
+static occa::kernel expansionCoeffKernel;
+occa::memory o_expansionCoeff;
 
 static dfloat P_DELTA;
 static dfloat P_GAMMA;
 
 void qtl(nrs_t* nrs, dfloat time, occa::memory o_div)
 {
+// ideal gas
   lowMach::qThermalIdealGasSingleComponent(time, o_div);
+
+// real gas
+//  expansionCoeffKernel(mesh->Nelements, cds->o_S, o_expansionCoeff);
```

Comment:
Simply use linAlg kernel to compute `1/T`. I think it's obvious that you can compute `o_expansionCoeff` with a user-supplied kernel as well.